apps/docs/src/getting-started.md

```diff
@@ -18,6 +18,22 @@ pnpm add fuels
 
 :::
 
+### Note
+
+If you are using bun, you'll need to add a `trustedDependencies` field to your `package.json`:
```

Comment:
We should mention `bun` in the docs only if the SDK fully supports it — not _only_ the CLI tool.

Reply:
@arboleya Great point. I have tried initializing and running the template with `bun`, and everything worked fine. But I'm not sure how to _fully_ verify if _everything_ in the SDK works with `bun`. Do you have any ideas?

Reply:
Can we remove all `bun` mentions for now and validate the support for it in another issue in the future? It will be safer than assuming anything (I have only assumptions now).

Maybe we can leave only this `NOTE` in the _Getting Started_ section and use the opportunity to reinforce we don't officially support it while removing all other shell command examples using `bun`.

Makes sense?

Reply:
That sounds fair. I'm removing the mentions of yarn and bun from the docs, and I've also removed them from the package mangers' list in the CLI, but I've still left behind the code that extracts the proper bin path in case someone still uses bun through some other way.